Vwitter/SpotifyAPI/SpotifyAPIManager.h

```diff
@@ -0,0 +1,37 @@
+//
+//  SpotifyAPIManager.h
+//  Vwitter
+//
+//  Created by Christina Li on 7/28/22.
+//
+
+#import <UIKit/UIKit.h>
+#import <SpotifyiOS/SpotifyiOS.h>
+
+NS_ASSUME_NONNULL_BEGIN
+typedef void (^SpotifyTrackCompletion)(NSMutableArray *_Nullable results, NSError *_Nullable error);
+
+typedef void (^DurationCompletion)(NSUInteger duration, NSError *_Nullable error);
+
+@interface SpotifyAPIManager : NSObject <SPTAppRemoteDelegate, SPTAppRemotePlayerStateDelegate>
+
+@property (nonatomic) SPTSessionManager *sessionManager;
+@property (strong, nonatomic) SPTAppRemote *appRemote;
+
+- (void)didAuthorizeWithSpotify:(NSString *)accessToken expiresInSeconds:(NSNumber *)expiresInSeconds;
+
++ (instancetype)shared;
```

Comment:
nit: usually initializers/singleton accessor methods like `shared` should be the first methods in a file, and something like `didAuthorizeWithSpotify` would be after those